atomisticparsers/lammps/parser.py

```diff
@@ -780,22 +781,35 @@ def get_traj_files(self):
         return [os.path.join(self.maindir, f) for f in traj_files]
 
     def get_data_files(self):
+        def check_file_header(file_path, regex_pattern):
+            header_size = 1024
+            file_path = f"{self.maindir}/{file_path}"
+            try:
+                with open(file_path, "rb") as file:
+                    file_header = file.read(header_size)
+                    file_header_str = file_header.decode(errors="ignore")
+            except FileExistsError:
```

Comment:
maybe put the general Exception here to include the decoding part?

Reply:
you mean just:
```
            except Exception:
                file_header_str = ""
```
Not sure what you mean by "to include the decoding part?